recipes/autopep8/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.2.4" %}
+package:
+  name: autopep8
+  version: {{ version }}
+
+source:
+  fn: autopep8-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/a/autopep8/autopep8-1.2.4.tar.gz
+  md5: fcea19c0c5e505b425e2a78afb771f5c
+
+build:
+  entry_points:
+    - autopep8 = autopep8:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pep8 >=1.5.7
+
+  run:
+    - python
+    - pep8 >=1.5.7
+
+test:
+  commands:
+    - autopep8 --help
+    - autopep8 --version
+
+about:
+  home: https://github.com/hhatto/autopep8
+  license: MIT License
```

Comment:
I usually remove the extra word License there as the field is already named "license." (No need to change to get this merged though!)